atlasdb-client/src/main/java/com/palantir/atlasdb/util/SlidingWindowMeanGauge.java

```diff
@@ -39,6 +39,7 @@ public void update(long entry) {
         histogram.update(entry);
     }
 
+    @SuppressWarnings("VisibleForTestingPackagePrivate") // used in SweepMetricsAssert
```

Comment:
As above - `VisibleForTesting` does nothing other than let the developer know, so surely may as well just drop a comment instead?